sprintly/sprintly-kanban
# Item detail updates

#### What's this PR do?
Formats links in the markdown formatter of the activity item.

#### How should this be manually tested?
Create a comment and see the link displayed in the activity item.
Same link parsing functionality used in the description to cover another ticket

#### What are the relevant tickets?
[436](https://kanban.sprint.ly/product/31528/item/436)
[486](https://kanban.sprint.ly/product/31528/item/486)

#### Screenshots (if appropriate)
##### Item Activity - [436]
![](http://g.recordit.co/p45xUG4lRp.gif)
##### Item Description - [486]
![image](https://cloud.githubusercontent.com/assets/2892213/10200675/9dac87a6-675c-11e5-8a37-5a19b8f8645e.png)
